src/spyglass/spikesorting/analysis/v1/group.py

```diff
@@ -189,8 +189,17 @@ def fetch_spike_data(
             ]
 
             # filter the spike times based on the labels if present
-            if "label" in nwb_file[nwb_field_name]:
-                group_label_list = nwb_file[nwb_field_name]["label"].to_list()
+            group_labels = getattr(
+                nwb_file[nwb_field_name],
+                "label",  # v0 compatibility
+                getattr(
+                    nwb_file[nwb_field_name],
+                    "curation_label",  # v1 compatibility
+                    None,
+                ),
+            )
```

Comment:
If we expect to see more cases in the future, i might convert to a for loop for readability:
```suggestion
            for attr in ("curation_label", "label"):  # v1, v0. if both, use v0
                label = getattr(nwb_file[nwb_field_name], attr, None)
```